cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -619,10 +619,20 @@ For services, you can retrieve:
 
 For workflows, you can retrieve:
 
-- the information about the specific run currently running::
```

Comment:
My question is: is it always a single current node? Because we have forks in the workflow, there can be more than one node currently executing.